lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -89,6 +90,7 @@ fb_picker.file_browser = function(opts)
   opts.custom_prompt_title = opts.prompt_title ~= nil
   opts.custom_results_title = opts.results_title ~= nil
   opts.use_fd = vim.F.if_nil(opts.use_fd, true)
+  opts.git_status = vim.F.if_nil(opts.git_status, true)
```

Comment:
Probably want to check `vim.fn.executable "git"` just in case someone doesn't have `git` now that this option will be true by default.

And add `git` as an optional dependency in the readme.

Reply:
I added and updated the documentation for this, do you want to give it another look?